mitodl/open-discussions
# Changed article thumbnail rendering to serve from embedly

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1745

#### What's this PR do?
- Uses embedly to serve thumbnails for our article cover images (like we've already been doing for link post thumbnails)
- Gets rid of old logic to generate article thumbnail on the fly, and get rid of the DB column that stored the image (the main cover image src is used instead to generate the thumbnail via embedly)

#### How should this be manually tested?
- Add `OPEN_DISCUSSIONS_USE_S3=True` in your `.env` file
- Also `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_STORAGE_BUCKET_NAME` in your `.env`. You can get the appropriate values from Heroku CI
- Create an article post and upload a cover image. The thumbnail for that post on a post listing should look correct at all screen widths.
- Create a link post. Same deal as above for the link preview thumbnail.

#### Any background context you want to provide?
- Old uploaded images for articles will have broken thumbnail images since they were previously saved to the local FS. This won't be a problem on prod since article images have always been uploaded to S3

#### Screenshots (if appropriate)
![ss 2019-02-05 at 16 34 57](https://user-images.githubusercontent.com/14932219/52376269-9ef1ff80-2a2f-11e9-93ef-e8955875452c.png)
![ss 2019-02-05 at 16 35 08](https://user-images.githubusercontent.com/14932219/52376254-913c7a00-2a2f-11e9-8781-031b701d6352.png)
![ss 2019-02-06 at 16 53 50](https://user-images.githubusercontent.com/14932219/52376345-e4163180-2a2f-11e9-975a-612f6d778506.png)
